docs/architecture/grpc-for-wcf-developers/client-libraries.md

```diff
@@ -142,7 +142,7 @@ namespace Grpc.Core
 
 ## Summary
 
-The `IAsyncEnumerable` and `IObservable` models are both well-supported and well-documented ways of dealing with asynchronous streams of data in .NET. gRPC streams map well to both paradigms, offering close integration with the modern .NET Core framework and reactive/asynchronous programming styles.
+The `IAsyncEnumerable` and `IObservable` models are both well-supported and well-documented ways of dealing with asynchronous streams of data in .NET. gRPC streams map well to both paradigms, offering close integration with the .NET Core framework, and reactive and asynchronous programming styles.
```

Comment:
I think this can be simplified 

```suggestion
The `IAsyncEnumerable` and `IObservable` models are both well-supported and well-documented ways of dealing with asynchronous streams of data in .NET. gRPC streams map well to both paradigms, offering close integration with .NET Core, and reactive and asynchronous programming styles.
```